### PR TITLE
Add sets to api

### PR DESCRIPTION
**Pull request by chasehensel**
@awans

### CODE REVIEW

File: internal/api/operations/create.go [hunk 1]
```diff
@@ -79,6 +79,32 @@ func (op NestedDisconnectOperation) ApplyNested(tx db.RWTx, parent db.ModelRef,
...
+func (op NestedSetOperation) ApplyNested(tx db.RWTx, parent db.ModelRef, parents []*db.QueryResult) (err error) {
+	outs, _ := handleRelationshipWhere(tx, parent, parents, op.Relationship, op.Where)
+
+	if len(outs) > 1 {
+		return fmt.Errorf("Found more than one record")
+	} else if len(outs) == 1 {
+		rec := outs[0].Record
+		//disconnect the old stuff
+		if op.Relationship.Multi() {
+			olds, _ := tx.GetRelatedManyReverse(rec.ID(), op.Relationship.ID())
```
Comment (awans) [inline internal/api/operations/create.go:92]: I think this is meant to use op.Relationship.LoadMany, and that should let us keep GetRelatedMany private to DB

Comment (chasehensel) [reply internal/api/operations/create.go:92]: I actually need the Reverses which is why I did that. I can instead create LoadManyReverse and LoadOneReverse?

Comment (awans) [reply internal/api/operations/create.go:92]: Let's do that! Sounds good.

**Comment by chasehensel** [comment]
@awans ready for another look!

**Pull request merged**